<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALF Visualizer</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="user_interaction_part" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="atoms_to_plot_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <widget class="QCheckBox" name="checkBox">
              <property name="geometry">
               <rect>
                <x>150</x>
                <y>80</y>
                <width>70</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="checkBox_2">
              <property name="geometry">
               <rect>
                <x>280</x>
                <y>170</y>
                <width>70</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="add_circle">
             <property name="text">
              <string>Add circle</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove_circle">
             <property name="text">
              <string>Remove Circle</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove_all">
             <property name="text">
              <string>Remove All</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove_grid">
             <property name="text">
              <string>Remove Grid</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="show_grid">
             <property name="text">
              <string>Show Grid</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="visibility_on_button">
             <property name="text">
              <string>Visibility On</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="visibility_off_button">
             <property name="text">
              <string>Visibility Off</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="change_sphere_color">
             <property name="text">
              <string>Change spherer to red</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pyvista_part" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="pyvista_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
